lib/js/src/manager/screen/choiceset/_PreloadPresentChoicesOperation.js

```diff
@@ -703,6 +703,9 @@ class _PreloadPresentChoicesOperation extends _Task {
             this._completionListener(this._loadedCells, success);
         }
 
+        // detach the OnKeyboardInput listener at the end
+        this._lifecycleManager.removeRpcListener(FunctionID.OnKeyboardInput, this._keyboardRpcListener);
```

Comment:
Removing the listener should be done after checking that the lifecycleManager is not null.